<?xml version="1.0"?>
<launch>

  <!-- Planificador -->
  <node pkg="planificacion_pkg" name="Secuenciador" type="Secuenciador" output="screen" launch-prefix="xterm -e"/>
  <node pkg="planificacion_pkg" name="prueba_movimiento" type="prueba_movimiento" output="screen" launch-prefix="xterm -e"/>
  <node pkg="planificacion_pkg" name="offset_camino" type="offset_camino"/>

</launch>
